moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -1305,6 +1305,15 @@ export const ProjectActivityLogOperationMaps = {
   },
 };
 
+export const ProjectActivityLogGenericDescriptions = {
+  project_extent_ids: {
+    label: "Project extent updated",
+  },
+  project_extent_geojson: {
+    label: "Project geo-json updated",
```

Comment:
Maybe this should be GeoJSON instead of geo-json?

Reply:
Good call, will change now